File: src/main/java/play/pluv/playlist/domain/PlayList.java
package play.pluv.playlist.domain;

import lombok.Builder;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

@Builder
@ToString
@Getter
@RequiredArgsConstructor
public class PlayList {

  private final PlayListId playListId;
  private final String name;
  private final String thumbNailUrl;
  private final Integer songCount;
}

File: src/main/kotlin/play/pluv/playlist/domain/PlayList.kt
package play.pluv.playlist.domain

data class PlayList(
    val playListId: PlayListId,
    val name: String,
    val thumbNailUrl: String,
    val songCount: Int? = null
) {

    //TODO: 호출부가 모두 kotlin으로 바꾸면 없얘기
    constructor(playListId: PlayListId, name:String, thumbNailUrl: String) : this(
        playListId = playListId,
        name = name,
        thumbNailUrl = thumbNailUrl,
        songCount = null
    )
}